utils/oscap-oval.c

```diff
@@ -363,12 +362,12 @@ int app_evaluate_oval(const struct oscap_action *action)
 
 	/* evaluation */
 	if (action->id) {
-		if ((oval_session_evaluate_id(session, action->probe_root, action->id, &eval_result)) != 0)
+		if ((oval_session_evaluate_id(session, NULL, action->id, &eval_result)) != 0)
 			goto cleanup;
 		printf("Definition %s: %s\n", action->id, oval_result_get_text(eval_result));
 	}
 	else {
-		if ((oval_session_evaluate(session, action->probe_root, app_oval_callback, NULL)) != 0)
+		if ((oval_session_evaluate(session, NULL, app_oval_callback, NULL)) != 0)
```

Comment:
You should IMO also refactor `oval_session_evaluate`, do we need probe_root as an arg there?

Reply:
We don't need it. But I think it would break the API,

Reply:
aha